Unix/wsman/wsmanclient.c

```diff
@@ -143,12 +145,23 @@ static void HttpClientCallbackOnStatusFn2(
         const ZChar *text)
 {
     WsmanClient *self = (WsmanClient*) callbackData;
-    if (!self->enumerationState || self->enumerationState->endOfSequence)
+
+    
+    if (result == MI_RESULT_TIME_OUT)
+        result = MI_RESULT_INVALID_PARAMETER;
```

Comment:
should be MI_RESULT_FAILED WITH an OMI_Error object, and set probable cause to 111 - timeout.